### PR TITLE
Template: add option to exclude documentation from pipeline template

### PR DESCRIPTION
**Pull request by mirpedrol**
Part of https://github.com/nf-core/tools/issues/2340 and https://github.com/nf-core/tools/issues/2342

### CODE REVIEW

**Review by mashehu — COMMENTED** [review]
hmm, shouldn't this just be `nf-core` because the main link in the readme is to the nf-core website?

**Comment by mirpedrol** [comment]
The links in the readme are actually only nf-core (see [here](https://github.com/nf-core/tools/pull/3130/files#diff-411c74708ad3051f07148b5c86c81577579469c9abaf96c8ed0163ab98a5edd2R85)). But the documentation files could be added for non-nf-core pipelines too, no?

**Comment by mashehu** [comment]
imo readme should always be there, because that is git-best-practices. the other two files should be optional

**Comment by mirpedrol** [comment]
The pipeline readme is always there 🙂 here we are only removing the readme inside `docs`

File: nf_core/pipelines/create/templatefeatures.yml [hunk 1]
```diff
@@ -300,3 +300,23 @@ slackreport:
...
   custom_pipelines: True
+documentation:
+  skippable_paths:
+    - "docs"
+  short_description: "Add documentation"
+  description: "Add documentation to the pipeline"
+  help_text: |
+    This will add documentation markdown files where you can describe your pipeline.
+    It includes:
+    - README.md: A README file where you can describe the basic use of your pipeline
```
Comment (mashehu) [inline nf_core/pipelines/create/templatefeatures.yml:311]:
```suggestion
    - docs/README.md: A README file where you can describe the structure of your documentation.
```

File: nf_core/pipelines/create/templatefeatures.yml [hunk 2]
```diff
@@ -300,3 +300,23 @@ slackreport:
...
+    - output.md: A file where you can explain the output generated by the pipeline
```
Comment (mashehu) [inline nf_core/pipelines/create/templatefeatures.yml:312]:
```suggestion
    - docs/output.md: A file where you can explain the output generated by the pipeline
```

File: nf_core/pipelines/create/templatefeatures.yml [hunk 3]
```diff
@@ -300,3 +300,23 @@ slackreport:
...
+    - usage.md: A file where you can explain the usage of the pipeline and its parameters.
```
Comment (mashehu) [inline nf_core/pipelines/create/templatefeatures.yml:313]:
```suggestion
    - docs/usage.md: A file where you can explain the usage of the pipeline and its parameters.
```

**Review by mashehu — APPROVED** [review]

**Comment by mashehu** [comment]
ah, was confused by the description. fixed that now

**Comment by mirpedrol** [comment]
thanks!

**Pull request merged**